src/Microsoft.DotNet.ImageBuilder/src/OctokitClientFactory.cs

```diff
@@ -74,14 +74,15 @@ information about the App itself.
                 var appClient = CreateClient(appCredentials);
                 var appInfo = await GetCurrentAppInfoAsync(appClient.Credentials);
 
-                Installation? appInstallation = appInfo.Installations.FirstOrDefault()
-                    ?? throw new InvalidOperationException(
-                        $"""
-                        No installations found for GitHub App.
-                        Ensure the App is installed on the organization you are trying to access.
-
-                        {appInfo}
-                        """);
+                Installation? appInstallation;
```

Comment:
We can remove the nullable annotation here:

```suggestion
                Installation appInstallation;
```